### PR TITLE
Managing OSTree content in Content Management Guide

### PR DESCRIPTION
**Pull request by Lennonka**
For Katello v4.3

Cherry-pick into:

* [x] Foreman 3.1

### CODE REVIEW

File: guides/common/modules/con_managing-ostree-content.adoc
Comment (melcorr) [inline guides/common/modules/con_managing-ostree-content.adoc:6]: I feel that you don't necessarily need to answer this question. 
We don't need to convince anyone to manage OSTree content in Katello. 
We just need to tell them how.

File: guides/common/modules/proc_managing-ostree-content-with-content-views.adoc
Comment (melcorr) [inline guides/common/modules/proc_managing-ostree-content-with-content-views.adoc:9]: I suppose you would need available repositories of synced OSTree content? 
For more info see Importing OSTree content? 
Something like that?

**Review by melcorr — COMMENTED** [review]
I am as happy as I can be with this. 
This is more than is available to folks at the moment. 
I'd happily merge. 
If you want some engineers to take a look, tag them!

File: guides/common/modules/proc_importing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_importing-ostree-content.adoc:57]:
```suggestion
```

File: guides/common/modules/proc_importing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_importing-ostree-content.adoc:45]:
```suggestion
```

File: guides/common/modules/proc_managing-ostree-content-with-content-views.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_managing-ostree-content-with-content-views.adoc:28]:
```suggestion
```

File: guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc:15]: Once this is no longer a draft, these comments should be removed.

File: guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc:40]:
```suggestion
* https://www.osbuild.org/guides/user-guide/building-ostree-images.html[Building OSTree images using the OSBuild tool]
```

`link:` should not be necessary.

**Review by maximiliankolb — COMMENTED** [review]
Nitpicking:

In general, you don't need a newline betwee `.Procedure` and `. My first bullet point` (same goes for prerequisites). Also, after a multi-line bullet point, you don't need a `+` before the next one.

File: guides/common/modules/con_managing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_managing-ostree-content.adoc:15]:
```suggestion
+
```

Note that the source code block is not part of the list.

File: guides/common/modules/proc_importing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_importing-ostree-content.adoc:10]: We have an attribute for that: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L87

File: guides/common/modules/proc_importing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_importing-ostree-content.adoc:18]: This is also optional.

File: guides/common/modules/proc_importing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_importing-ostree-content.adoc:26]: This is probably also optional.

File: guides/common/modules/proc_importing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_importing-ostree-content.adoc:41]: I am unsure about the capitalization of `Synchronization Status`. I've found it twice on the master branch all lowercase.

File: guides/common/modules/proc_importing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_importing-ostree-content.adoc:25]:
```suggestion
For example `\http://www.example.com/rpm-ostree/`.
```

Maybe we should escape the link? example.com/anything should never be clicked/clickable.

File: guides/common/modules/proc_managing-ostree-content-with-content-views.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_managing-ostree-content-with-content-views.adoc:11]: Please use the proper attribute.

File: guides/common/modules/proc_managing-ostree-content-with-content-views.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_managing-ostree-content-with-content-views.adoc:17]:
```suggestion
. Optional: If this Content View is for importing from an upstream server only, select the *Import only* check box.
You cannot publish import-only Content Views directly.
```

File: guides/common/modules/proc_managing-ostree-content-with-content-views.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_managing-ostree-content-with-content-views.adoc:45]:
```suggestion
. Publish the Content View:
```

We have mixed capitalizations on the master branch. @melcorr's call.

File: guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc:8]:
```suggestion
You can also create an OSTree image archive by downloading a repository from an upstream URL, such as:

```

File: guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc:11]:
```suggestion
# wget --no-parent -r _https://repos.example.com/ostree/repo_name/_
```

Please prefix both commands.

File: guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc:24]:
```suggestion
# hammer repository upload-content --id _repository_id_ \
```

File: guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc
Comment (maximiliankolb) [inline guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc:33]: Can you please make this line non-orcharhino only?

File: guides/common/modules/proc_managing-ostree-content-with-content-views.adoc
Comment (melcorr) [reply guides/common/modules/proc_managing-ostree-content-with-content-views.adoc:45]: It should always be Content View. I might ask @AkshayGadhaveRH to look for instances of this being wrong on master branch and correct it for us as an exercise. @AkshayGadhaveRH would you be willing to try this?

File: guides/common/modules/con_managing-ostree-content.adoc
Comment (jlsherrill) [inline guides/common/modules/con_managing-ostree-content.adoc:14]: You might mention that it is el8 only.  (doesn't work on el7)

File: guides/common/modules/proc_managing-ostree-content-with-content-views.adoc
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_managing-ostree-content-with-content-views.adoc:45]: @melcorr on it

File: guides/common/modules/con_managing-ostree-content.adoc
Comment (Lennonka) [reply guides/common/modules/con_managing-ostree-content.adoc:14]: Thanks for catching that! I intended, but forgot :)

File: guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc
Comment (Lennonka) [reply guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc:33]: Sure!
Thanks for all your catches. On it.

File: guides/common/modules/proc_managing-ostree-content-with-content-views.adoc
Comment (Lennonka) [reply guides/common/modules/proc_managing-ostree-content-with-content-views.adoc:45]: @AkshayGadhaveRH I'm going to correct this one and you can make another PR to fix it elsewhere. Agreed?

File: guides/common/modules/proc_importing-ostree-content.adoc
Comment (Lennonka) [reply guides/common/modules/proc_importing-ostree-content.adoc:41]: I was unsure too, gonna lowercase it.

File: guides/common/modules/proc_managing-ostree-content-with-content-views.adoc
Comment (melcorr) [reply guides/common/modules/proc_managing-ostree-content-with-content-views.adoc:45]: Yes. Done in #1050 -rebase to enjoy

File: guides/common/modules/con_managing-ostree-content.adoc
Comment (Lennonka) [inline guides/common/modules/con_managing-ostree-content.adoc:13]: @jlsherrill @melcorr @maximiliankolb Are we okay with the phrasing here?

Comment (Lennonka) [reply guides/common/modules/con_managing-ostree-content.adoc]: Maybe now it's better.

Comment (melcorr) [reply guides/common/modules/con_managing-ostree-content.adoc]: Fine by me!

**Comment by Lennonka** [comment]
FYI, we're waiting for one more review by @jlsherrill because we may want to disable some parts for Satellite 7.0 as we're a bit ahead with docs for this release. :)

File: guides/common/assembly_managing-ostree-content.adoc
Comment (melcorr) [reply guides/common/assembly_managing-ostree-content.adoc:5]: @jlsherrill we are close to branching for 3.2, should we split this and not cherry pick for 3.1 or will this just never be relevant to Satellite?

Comment (jlsherrill) [reply guides/common/assembly_managing-ostree-content.adoc:5]: it will be relevant in the future, but not something that is part of the intended use case today

Comment (jlsherrill) [reply guides/common/assembly_managing-ostree-content.adoc:5]: I'm not sure i follow about the choices here 'split this and not cherry pick for 3.1'

Comment (jlsherrill) [reply guides/common/assembly_managing-ostree-content.adoc:5]: everything else look great BTW!

**Comment by Lennonka** [comment]
@melcorr Thank you! It should be ready for merge.

File: guides/common/modules/con_managing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_managing-ostree-content.adoc:13]:
```suggestion
* OSTree content is supported only when running {Project} on {RHEL} 8.
```

Does is also work with CentOS 8? See https://docs.theforeman.org/nightly/Installing_Server_on_Red_Hat/index-foreman-el.html#supported-operating-systems_foreman

File: guides/common/modules/con_managing-ostree-content.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_managing-ostree-content.adoc:21]:
```suggestion
* If you want to use Hammer CLI, you must enable the `hammer-cli-katello` plugin on {ProjectServer}.
```

**Review by maximiliankolb — APPROVED** [review]
Two small suggestions.
The rest LGTM!

File: guides/common/modules/con_managing-ostree-content.adoc
Comment (Lennonka) [reply guides/common/modules/con_managing-ostree-content.adoc]: It actually does (confirmed with devs)! Good catch. Please check if you're okay with the condition.

**Comment by melcorr** [comment]
Cherry-picked to 3.1 
   89b02285..e6209563  3.1 -> 3.1

**Pull request merged**
